update_model.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
Remove these artifacts (L1-L4) that came from converting a jupyter cell to py script, just start the top of the script with a description of the file:

you can use:
https://github.com/pnnl-predictive-phenomics/SElon-GEM/blob/main/syn_elong/model_changes.py

as an example